README.md

```diff
@@ -41,7 +41,7 @@ When pulling code changes from git, be sure to update your local packages from r
 
 ## Configuration Settings
 
-Default settings are loaded from `defaults.config`. To override these settings, create a configuration file and save the path to this file in the environment variable `PRINTAPP_SETTINGS`.
+Default settings are loaded from `src/printapp/config.py`. To override these settings, create a configuration file and save the path to this file (relative to the `src/printapp` directory) in the environment variable `PRINTAPP_SETTINGS`.
```

Comment:
It is actually not relative to where you launch the app from. I just tested this. It is relative to `__init__.py`. But using absolute paths would be best.